### PR TITLE
feat(studio): Comparison Line Chart common component

### PR DESCRIPTION
**Pull request by steramae-nvidia**
Screenshot 2026-08-11 at 2 52 25 PM
Screenshot 2026-08-11 at 2 52 06 PM

feat(studio): Comparison Line Chart common component

Signed-off-by: Sean Teramae <steramae@nvidia.com>

lock file

Signed-off-by: Sean Teramae <steramae@nvidia.com>

## Type of Change

- [ ] Code change (feature, bug fix, or refactor)
- [ ] Code change with documentation updates
- [ ] Documentation only
- [ ] Contributor tooling or automation
- [ ] CI, build, or test infrastructure

## Quality Gates

- [ ] Tests added or updated for changed behavior
- [ ] Existing tests cover changed behavior — justification:
- [ ] Tests not applicable — justification:
- [ ] Documentation updated for user-visible behavior
- [ ] Documentation not applicable — justification:

## Verification

- [ ] Pull request title follows the repository's Conventional Commit format
- [ ] Every commit includes an appropriate `Signed-off-by:` trailer
- [ ] `uv run pre-commit run -a` passes, or any blocked checks are identified below
- [ ] Targeted tests pass, or tests are marked not applicable above
- [ ] No secrets, API keys, or credentials are included

Targeted validation:

## Summary by CodeRabbit

* **New Features**
  * Added a reusable comparison line chart with configurable axes, series, legends, tooltips, annotations, reference lines, and formatting.
  * Added interactive series visibility, hover highlighting, and dashed reference series.
  * Added loading and empty states.
  * Added support for time, numeric, and category-based comparisons.

* **Tests**
  * Added comprehensive coverage for rendering, interactions, formatting, annotations, and edge cases.

* **Chores**
  * Excluded generated Storybook output from version control.

### CODE REVIEW

**Comment by steramae-nvidia** [comment]
This change is part of the following stack:

- #1244 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

File: web/packages/common/src/components/ComparisonLineChart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+const TICK_STYLE = { fontSize: 11, fill: AXIS_TEXT_COLOR } as const;
+const AXIS_LABEL_STYLE = { fontSize: 12, fill: AXIS_TEXT_COLOR } as const;
+
+/**
+ * Multi-series line chart for comparing runs, models, or variants over a shared x axis.
+ * Series are colored from the shared palette, the legend toggles them on and off, and hovering a
+ * legend entry fades the others so a single line can be read out of a crowded chart.
+ */
+export const ComparisonLineChart = ({
```
Comment (marcusds) [inline web/packages/common/src/components/ComparisonLineChart/index.tsx:39]: This component file is really large, is it possible to split some of the sub components out or anything?

**Review by marcusds — APPROVED** [review]

File: web/packages/common/src/components/ComparisonLineChart/index.tsx (hunk 1 above)
Comment (steramae-nvidia) [reply web/packages/common/src/components/ComparisonLineChart/index.tsx:39]: will try!
